src/components/DocumentViewer/PDFViewer.js

```diff
@@ -1,6 +1,6 @@
 import React, { useContext, useEffect, useRef, useState } from "react";
 import { DocumentViewerContext } from "./DocumentViewerContext";
-import { decodeBase64 } from "../../util/Base64Util";
+import { decodeBase64 } from "../../util/base64Util";
```

Comment:
Note that I fixed the casing. Utils should be camelCase since they're not instantiated like React components.

Reply:
Renamed